types/sequelize/sequelize-tests.ts

```diff
@@ -1055,6 +1055,7 @@ findOrRetVal = User.findOrCreate( { where : { email : 'unique.email.@d.com', com
 findOrRetVal = User.findOrCreate( { where : { objectId : 1 }, defaults : { bool : false } } );
 
 let upsertPromiseNoOptions: Bluebird<boolean> = User.upsert( { id : 42, username : 'doe', foo : s.fn( 'upper', 'mixedCase2' ) } );
+let upsertPromiseWithNonReturningOptions: Bluebird<boolean> = User.upsert( { id : 42, username : 'doe', foo : s.fn( 'upper', 'mixedCase2' ) }, { logging: true } );
```

Comment:
The return type `Bluebird<boolean>` should not pass the test if the function does not return anything.

Reply:
@oktapodia im not sure what you mean here, as per the sequelize docs, `upsert` returns a Promise<boolean> _unless_ the `returning` option is true, then it returns a Promise<Model, created>.

See http://docs.sequelizejs.com/class/lib/model.js~Model.html#static-method-upsert

It might be the name of the value being defined here `upsertPromiseWithNonReturningOptions` that is confusing?  It is intended to communicate that the options passed did not include a `returning` option.  Is that the issue? If so, do you have a suggestion for a name?

Reply:
my bad sorry!